timescaledb/how-to-guides/compression/decompress-chunks.md

```diff
@@ -28,8 +28,9 @@ SELECT tableoid::regclass FROM metrics
 ```
 
 Decompression might often be employed in the event that you need to backfill or
-update data that lives in a compressed chunk, as TimescaleDB does not currenty
-support modifying (inserting into, updating, deleted from) compressed chunks.
+update data that lives in a compressed chunk, as TimescaleDB only partially
+supports modifying (inserting into, not updating or deleting from) compressed
+chunks.
```

Comment:
Inserting into compressed chunks is more expensive when compared to inserting into an uncompressed chunk - you might want to consider decompressing the chunk before backfilling huge amounts of data